netkan/netkan/metadata.py

```diff
@@ -94,14 +95,27 @@ class Ckan:
         'application/x-compressed-tar': 'tar.gz',
         'application/zip':              'zip',
     }
+    ISODATETIME_PROPERTIES = [
+        'release_date'
+    ]
 
     def __init__(self, filename=None, contents=None):
         if filename:
             self.filename = Path(filename)
             self.contents = self.filename.read_text()
         else:
             self.contents = contents
-        self._raw = json.loads(self.contents)
+        self._raw = json.loads(self.contents, object_hook=self._custom_parser)
+
+    def _custom_parser(self, dct):
+        # Special handling for DateTime fields
+        for k in self.ISODATETIME_PROPERTIES:
+            if k in dct:
+                try:
+                    dct[k] = dateutil.parser.isoparse(dct[k])
+                except:
+                    pass
```

Comment:
That's a pretty broad catch, probably ok here, but can lead to hiding errors we didn't expect.

Reply:
Yeah, that's intentional. Don't want the whole thing to come to a screeching halt because there's some problem parsing a date string. If that happens, it'll just leave the string in the dict, which is fine.